From 9c93e3a0e6b7306e22f9dd688890a5962b11a5d8 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:28:15 -0400
Subject: [alsa] sis7019: Simplify dependencies

Message-id: <140972823573-237-patchset-jkysela@redhat.com>
Patchwork-id: 90937
O-Subject: [RHEL 7.1 PATCH 236/442] ALSA: sis7019: Simplify dependencies
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

Kconfig symbol X86_32 was introduced in October 2005, it's about time
to use it :-)

Signed-off-by: Jean Delvare <jdelvare@suse.de>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 4c16ecc4c0715e0b6b3c7b72976c05b0e212cd0f)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/Kconfig b/sound/pci/Kconfig
index 3e22b28..98a4591 100644
--- a/sound/pci/Kconfig
+++ b/sound/pci/Kconfig
@@ -792,7 +792,7 @@ config SND_RME9652
 
 config SND_SIS7019
  tristate "SiS 7019 Audio Accelerator"
- depends on X86 && !X86_64
+ depends on X86_32
  select SND_AC97_CODEC
  select ZONE_DMA
  help
-- 
1.7.1